src/api/controllers/articleController.js

```diff
@@ -53,6 +52,110 @@ export default class ArticleController {
     }
   }
 
+  /**
+   * publish a drafted article
```

Comment:
Are we publishing or storing the draft? @alainburindi

Reply:
we are publishing